update.py

```diff
@@ -110,8 +110,7 @@ def importNuitka():
 from nuitka.utils.ReExecute import callExecProcess
 from nuitka.utils.Rest import makeTable
 
-in_devcontainer = os.getenv("REMOTE_CONTAINERS_DISPLAY_SOCK") is not None
-
+in_devcontainer = os.getenv("IN_DEVCONTAINER") == "1"
```

Comment:
Maybe we want to make it a command line option, so disabling becomes easier. Then only auto-update.sh would provide it, and we can switch it around easily. That seems more sensible. Lets call the option --development and rename this to `development_mode` and then try and keep that minimal over time.